css/themes/default/jquery.mobile.theme.css

```diff
@@ -611,3 +613,22 @@ button[disabled],
 	outline-width: 1px;
 	outline-style: inherit;
 }
+
+.ui-accordion-header {
+	background: #e6e6e6;
+	border: 1px solid #d3d3d3;
+	color: #555555;
+}
+
+.ui-accordion-content {
```

Comment:
this should be added to https://github.com/jquery/jquery-mobile/blob/1.5-dev/css/themes/default/jquery.mobile.theme.css#L268 and https://github.com/jquery/jquery-mobile/blob/1.5-dev/css/themes/default/jquery.mobile.theme.css#L435 instead of creating a new rule i think